src/main/java/teammates/ui/webapi/action/BasicFeedbackSubmissionAction.java

```diff
@@ -67,10 +65,6 @@ protected StudentAttributes getStudentOfCourseFromRequest(String courseId) {
      */
     protected void checkAccessControlForStudentFeedbackSubmission(
             StudentAttributes student, FeedbackSessionAttributes feedbackSession) {
-        if (student == null) {
```

Comment:
If we remove the null check, will it results in NPE? (Line 74)

Reply:
I believe so. Now I see why there is a need for the null check to be there.

In that case, I will change the exception thrown by the null check to be `UnauthorizedAccessException` instead of `EntityNotFoundException`